source/frontend/src/features/properties/map/shared/models.ts

```diff
@@ -1,10 +1,14 @@
 import { IMapProperty } from 'components/propertySelector/models';
+import { IBcAssessmentSummary } from 'hooks/useBcAssessmentLayer';
+import { Api_Address } from 'models/api/Address';
 import { Api_File } from 'models/api/File';
 import { Api_Property } from 'models/api/Property';
 import { Api_PropertyFile } from 'models/api/PropertyFile';
 import { formatApiAddress, pidParser } from 'utils';
 import { toTypeCode } from 'utils/formUtils';
 
+import { formatBcaAddress } from './../../../../utils/propertyUtils';
```

Comment:
nit: would be a bit better to include this import in the 'utils' import 2 lines above